test/06_cross-chain_minting_tests_l1_l2.ts

```diff
@@ -399,7 +359,7 @@ describe('Testing cross-chain minting (L1 & L2)', async function () {
         // temporarily set MockLZEndpoint as messaging module, to allow for easy sending
         await l2.operator.setMessagingModule(l2.mockLZEndpoint.address);
         // make call with mockLZEndpoint AS messaging module
-        await l2.mockLZEndpoint.crossChainMessage(l2.operator.address, getLzMsgGas(payload), payload, {
+        await l2.mockLZEndpoint.crossChainMessage(l2.operator.address, getLzMsgGas(String(payload)), payload, {
```

Comment:
for these existing tests, why was the conversion necessary?

Reply:
it was not necessary, but the linter was reporting errors.